setup.py

```diff
@@ -12,17 +12,12 @@
     "perfect-jsonschema",
     "colorama",
     "tqdm",
+    "ipywidgets",
 )
 dependency_links = ["https://github.com/ambv/black.git#egg=black"]
 
 
-tests_require = [
-    "pytest",
-    "pytest-mock",
-    "pytest-cov",
-    "pytest-pythonpath",
-    "ipywidgets",
-]
+tests_require = ["pytest", "pytest-mock", "pytest-cov", "pytest-pythonpath"]
```

Comment:
I do prefer it as before: one item per line. 

Seems like it's easier to maintain and when adding/removing items, the diffs look smaller.